docs/administration/configuration/configuring-access/index.md

```diff
@@ -24,8 +24,62 @@ Helm value accordingly.
 
 ## User registration via Single Sign-On
 
-Handling of users with Single Sign-On (SSO) is also possible. Currently,
-this is only available for CERN deployments via [`components.reana_ui.cern_sso`](https://github.com/reanahub/reana/tree/master/helm/reana)
+Handling of users with Single Sign-On (SSO) is also possible. Currently SSO is supported for CERN as well as any third-party SSO provider which supports [Keycloak](https://www.keycloak.org/).
+
+### Keycloak Single Sign-On Configuration
+
+First of all, to integrate REANA with your Keycloak instance, you need to create a new client from the Keycloak's admin dashboard.
+There are many configuration options, but these are the minimal changes you have to make:
+
+- _Valid Redirect URIs_ should be set to `https://my-reana-hostname/api/oauth/authorized/keycloak/` (see Keycloak's [Access Settings](https://www.keycloak.org/docs/latest/server_admin/#access-settings))
```

Comment:
```suggestion
- _Valid Redirect URIs_ should be set to `https://<your-reana-hostname>/api/oauth/authorized/keycloak/` (see Keycloak's [Access Settings](https://www.keycloak.org/docs/latest/server_admin/#access-settings))
```
I'd switch from "my reana hostname" to "your reana hostname", just to be consistent with the rest of the placeholders. What do you think?

Also, as a rule of thumb, in general, throughout the documentation, whenever we write a placeholder value with an explanatory name that will be replaced (such as "your reana hostname"), we surround it with `< >`, unless it's an example value: for example, when referring to an email address in an example, you would either write `johndoe@example.com` or `<your-email-address>`. The same observation is valid below for `your-provider-hostname`, `your-client-id`, `your-client-secret` and so on. See for example how it is done [here](https://docs.reana.io/administration/configuration/configuring-tls-certificates/) or [in this paragraph](https://docs.reana.io/administration/configuration/configuring-storage-volumes/#cephfs).

Reply:
I have changed it to `<your-reana-hostname>`, but please note that unfortunately we are far from consistent also in the rest of docs (e.g. `my` vs `your`, even in the pages you have linked, such as `mycert.crt` or `myvalues.yaml`). Actually, should we use `your-reana-hostname` or change the other URLs to have `my-provider-hostname`?

Reply:
I mainly meant consistency in the same page (even though of course extending it to the rest of the docs wouldn't be bad). I personally prefer the "your" version, especially since the documentation is written in the second person point of view ("you need to create..", "you can use...")